src/addons/action-button.js

```diff
@@ -24,11 +24,13 @@ class ActionButton extends React.PureComponent {
   }
 
   static defaultProps = {
-    tag: 'button'
+    tag: 'button',
+    prepare: (event, value) => value
   }
 
   click(event: Event): Action {
-    let action = this.send(this.props.action, this.props.value)
+    let payload = this.props.prepare(event, this.props.value)
```

Comment:
ActionForm's prepare is `(value) => value)`. For consistency, what do you think of doing `(value, event) => value` for both?